### PR TITLE
feat: add cli package #hacktoberfest

### PR DESCRIPTION
**Pull request by subekti404dev**
re-PR based on #5 🙏 
#hacktoberfest

### CODE REVIEW

File: packages/cli/src/index.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    jest.setTimeout(60000);
+  });
+
+  test('should generate json', async () => {
+    const id = `12kxsNITg_kHRCs7DUYUOqzj-tXzdO3r0nGUYMHvAQlo`;
+    const indexFilePath = path.join(__dirname, 'index.js');
+    const outputFilename = 'test-data.json';
+    const outputFilePath = path.join(__dirname, '..', outputFilename);
+    await execAsync(`node ${indexFilePath} pull --spreadsheet-id ${id} --out ${outputFilename}`);
+    expect(existsSync(outputFilePath)).toEqual(true);
```
Comment (afrianjunior) [inline packages/cli/src/index.test.js]: can you adjust this test to make sure the file meet expectations?

**Review by afrianjunior — CHANGES_REQUESTED** [review]

File: packages/cli/src/index.test.js (hunk 1 above)
Comment (subekti404dev) [reply packages/cli/src/index.test.js]: updated mas @afrianjunior 🙏 .

**Comment by r17x** [comment]
@subekti404dev  @afrianjunior  This PRs better merge until #13  was merged

File: package.json [hunk 2]
```diff
@@ -24,6 +24,9 @@
     "root": true,
     "extends": [
       "eword"
+    ],
+    "ignorePatterns": [
```
Comment (r17x) [inline package.json]: ini boleh dihapus mang ? 

jadi kalau mau updat test confignya mending didalam tooling/eslint

Comment (subekti404dev) [reply package.json]: tooling/eslint itu dimana ya? soalnya ini emang butuh di ignore sih .. kalau ga, gabisa dicommit

Comment (r17x) [reply package.json]: Di packages/eslint-config 

nah ada index.js. kamu bisa configure disitu.

Comment (subekti404dev) [reply package.json]: updated ya @r17x

Comment (r17x) [reply package.json]: Mantulity @subekti404dev

**Review by afrianjunior — APPROVED** [review]

**Comment by afrianjunior** [comment]
#hacktoberfest-accepted

**Pull request merged**